app/src/main/res/layout/component_autocomplete_textview.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
merge tag would work for now because this component is inside another RelativeLayout, but if we need to change parent structure, it will break.